File: fizz_buzz/fizz_buzz.java
public class Solution {
  public List<String> fizzBuzz(int n) {
    List<String> myList = new ArrayList<String>();
    for(int i = 1; i < n+1; i++){
      String str = "";
      if ((i % 3) == 0){
        str = str + "Fizz";
      }
      if ((i % 5) == 0){
        str = str + "Buzz";
      }
      if ((i % 3) != 0 && (i % 5) != 0){
        str = String.valueOf(i);
      }
      // String str = String.valueOf(i);
      myList.add(str);
    }
    // System.out.println(myList);
    return myList;
  }
}

File: fizz_buzz/fizz_buzz.rb
# @param {Integer} n
# @return {String[]}
def fizz_buzz(n)
  arr = []
  n.times{|n| arr<<(n+1).to_s}
  len = arr.length
  i = 0 
  while i < len
  	j = i + 1
  	if j % 3 == 0 || j % 5 == 0
  		if j % 3 == 0 && j % 5 == 0
  			arr[i] = 'FizzBuzz'
  		elsif j % 3 == 0
  			arr[i] = 'Fizz'
  		elsif j % 5 == 0
  			arr[i] = 'Buzz'
  		end
  	end
  	i += 1
  end
  arr
end